test/unit/specs/util/test-schemas.js

```diff
@@ -893,3 +893,98 @@ export const uniqueFieldSchema = {
     }
   ]
 }
+
+export const defaultStringValue = {
+  'attributes': [
+    {
+      'attributes': [],
+      'auto': false,
+      'fieldType': 'STRING',
+      'href': '/api/v2/sys_sec_User/meta/username',
+      'isAggregatable': false,
+      'label': 'Username',
+      'labelAttribute': true,
+      'lookupAttribute': true,
+      'maxLength': 255,
+      'name': 'username',
+      'nillable': false,
+      'readOnly': true,
+      'unique': true,
+      'visible': true,
+      'defaultValue': 'default string value'
+    }
+  ]
+}
+
+export const defaultBooleanValue = {
+  'attributes': [
+    {
+      'attributes': [],
+      'auto': false,
+      'fieldType': 'BOOL',
+      'href': '/api/v2/sys_sec_User/meta/username',
+      'isAggregatable': false,
+      'label': 'Username',
+      'labelAttribute': true,
+      'lookupAttribute': true,
+      'maxLength': 255,
+      'name': 'username',
+      'nillable': false,
+      'readOnly': true,
+      'unique': true,
+      'visible': true,
+      'defaultValue': true
```

Comment:
defaultValue always contains a string and the client should cast it to the proper type

Reply:
Why is it designed like this? , I think we should change this in the backend, but will change is for now.